README.md

```diff
@@ -36,7 +36,7 @@ Pre for `<pre>` block, code for `<code>` span snippet.
 
 Clone the repo:
 
-    $ git clone git@github.com:haskell-infra/hl.git
+    $ git clone https://github.com/haskell-infra/hl
```

Comment:
That depends on their workflow and also if they are in the haskell-infra org. I would keep it as it was for the git@ one, I think.